### ISSUE TITLE
New User - Issues submitting from files on another local drive

### ISSUE THREAD

**Issue by TheComputerGuyDR** [exercism/exercism#4509]
Hello,

I'm stepping back into programing and I'm starting with C#. I was able to download the Hello World File but when I finished I'm having issues Submitting it. The error I'm getting is with the syntax. I'm not sure if I need to be using "\" or "/" when submitting. If you can see the screenshot, I'm trying to get from my F drive so I'm not sure if I need to "cd" or change directory when uploading either. I've tried a few different ways and all getting different results but nothing submitting.

![screenshot_1](https://user-images.githubusercontent.com/45431645/49167348-d12e0500-f2fa-11e8-8155-30582b005a50.png)

**Comment by NobbZ** [comment]
You need to use `"` when your file path contains spaces. This is not related to exercism, but how arguments gets tokenized by the shell and passed to the program to run.

Also the path you passed to `exercism` doesn't look like it were in your `exercism workspace`. Please make sure you do your stuff in the workspace or move files back and forth between your editors workspace and exercisms. I strongly advice against the double bookkeeping necessary for the latter.

**Comment by TheComputerGuyDR** [comment]
That was helpful.
I was able to:
exercism submit "F:\WorkSpace\Visual Studio\csharp projects\Hello World\Hello World\Hello World\HelloWorld.cs"
That worked just fine.
What was throwing me off was this was posted how to upload:
exercism submit path/to/file1 [path/to/file2 ...]
So my thought was that I needed to upload two files instead of just doing one file.

I'm sure if there is multiple files I need to give paths to all the files but I can leave out the "[ ]" and just do the paths.
